tests/unit/modules/zypper_test.py

```diff
@@ -382,6 +387,22 @@ def test_upgrade_success(self):
     @patch('salt.modules.zypper.refresh_db', MagicMock(return_value=True))
     @patch('salt.modules.zypper._systemd_scope', MagicMock(return_value=False))
     @patch.dict('salt.modules.zypper.__grains__', {'osrelease_info': [12, 1]})
+    def test_upgrade_kernel(self):
+        '''
+        Test kernel package upgrade success.
+
+        :return:
+        '''
+
+        with patch.dict(zypper.__salt__, {'pkg_resource.parse_targets': MagicMock(return_value=(['kernel-default'], None))}):
+            with patch('salt.modules.zypper.__zypper__.noraise.call', MagicMock()):
+                with patch('salt.modules.zypper.list_pkgs', MagicMock(side_effect=[{"kernel-default": "3.12.49-11.1"}, {"kernel-default": "3.12.49-11.1,3.12.51-60.20.2"}])):
```

Comment:
And while we do not have that stupid 80 chars punch-card limit on width as we no longer on CRT IBM terminals, still we shouldn't abuse the freedom. :wink: So I'd keep it 120 for the sanity reasons.